zookeeper-server/src/test/java/org/apache/zookeeper/ClientCnxnSocketFragilityTest.java

```diff
@@ -0,0 +1,310 @@
+package org.apache.zookeeper;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.util.Queue;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
```

Comment:
Some imports are never used, also notice the useless field in this class(e.g: `scheduledExecutor`)

Reply:
Thanks for comment. Accept it. I miss the check for unused field. I will fix it.